test/frame.test.js

```diff
@@ -17,7 +17,8 @@ describe('Recurly.Frame', function () {
     });
 
     // Returns a spy so frame.bindWindowCloseListener does not cancel the tick
-    window.open = sinon.spy(() => sinon.spy());
+    const windowStub = { close: () => { windowStub.closed = true; }, closed: false };
```

Comment:
This gives `Frame.destroy` a `.close()` method to call, which is triggered when 'close' is emitted. It's circular and confusing, but it ensures the window actually is closed. Previously, the tests would exit early because the error would hit the ceiling, but the reporter would not bubble the failure properly.